templates/semantic_segmentation/2d_semantic_segmentation.yaml

```diff
@@ -1,49 +1,49 @@
-# BiaPy version: 3.5.13
+    # BiaPy version: 3.5.13
```

Comment:
Why are you changing the template? It is a mistake right? Please remove these changes